graph_asset_inventory_api/inventory/universe.py

```diff
@@ -0,0 +1,71 @@
+"""This modules provides the class ``CurrentUniverse`` that define the
+information about the current version of the asset inventory universe"""
+
+
+class UniverseVersion:
+    """Represents a version of a Universe"""
+
+    def __init__(self, version):
+        self.int_version = UniverseVersion.__int_version(version)
+        self.sem_version = version
+
+    @classmethod
+    def __int_version(cls, sem_version):
+        """Returns the int representation of the universe version used to
+        compare two semvers. The semver parameter must have the following
+        shape: xx.xx.xx where x is a digit. That is: the major, minor and patch
```

Comment:
I think it is okey to accept the limitation, because taking into account the pre-release and build metatata formats when converting the ``semver`` to an integer will make quite more complex this function, and I think the expresevity that this ``pseudo semver`` give us should be enough. In any case I'm completely agree I should document better that we are not using an actual sem version here, but a more constrained form of it.

Reply:
I'm adding documentation to the ``__init__`` function to document the shape of the version parameter.
I also removed all mentions to ``semver`` in order the avoid confusions.